File: delivery-system-server/src/main/java/ru/delivery/system/model/json/WarehousesInfoResponse.java
package ru.delivery.system.model.json;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

public class WarehousesInfoResponse {
    @JsonProperty("id")
    @Getter @Setter
    private Integer id;

    @JsonProperty("address")
    @Getter @Setter
    private String address;

    @JsonProperty("latitude")
    @Getter @Setter
    private Float latitude;

    @JsonProperty("longitude")
    @Getter @Setter
    private Float longitude;
}

File: delivery-system-desktop/src/main/kotlin/ru/delivery/system/models/json/WarehousesInfoResponse.kt
package ru.delivery.system.models.json

import com.fasterxml.jackson.annotation.JsonProperty

class WarehousesInfoResponse {
    @JsonProperty("id")
    var id: Int? = null

    @JsonProperty("address")
    var address: String? = null

    @JsonProperty("latitude")
    var latitude: Float? = null

    @JsonProperty("longitude")
    var longitude: Float? = null
}